lib/debugletapi.js

```diff
@@ -83,7 +83,7 @@ DebugletApi.prototype.init = function(uid, logger, callback) {
         logger.warn('Malformed source-context.json file.');
         // But we keep on going.
       }
-      callback(null);
+      callback(null, project);
```

Comment:
There are paths in this function where `callback` is called without the `project` parameter – those are actually the cases where we *need* to know the project number.

Matt's suggestion about the test below would catch this.